tests/handlebars-attr-node/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,30 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`brackets.hbs 1`] = `
+====================================options=====================================
+parsers: ["glimmer"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+<div title='My "title"'></div>
+<div title="My other 'title'"></div>
+
+=====================================output=====================================
+<div title='My "title"'></div><div title="My other 'title'"></div>
+================================================================================
+`;
+
+exports[`brackets.hbs 2`] = `
+====================================options=====================================
+parsers: ["glimmer"]
+printWidth: 80
+singleQuote: true
+                                                                                | printWidth
+=====================================input======================================
+<div title='My "title"'></div>
+<div title="My other 'title'"></div>
+
+=====================================output=====================================
+<div title='My "title"'></div><div title="My other 'title'"></div>
```

Comment:
I think we need quotes here like do prettier for js

Reply:
You mean, we should have for the second div?
```
<div title='My other \'title\''></div>
```

On master, hbs printer produces the following output (the two divs wrong, the first being very wrong):
```
<div title="My "title""></div><div title="My other 'title'"></div>
```

Reply:
hm, all fine, my mistake